### ISSUE TITLE
Anyone tried Big Sur ?

### ISSUE THREAD

**Issue by alfonsodev** [banhbaoxamlan/X230-Hackintosh#32]
has any one tried with Big Sur Mac OS 11, 
do you know what are the challenges ?

**Comment by mave-panda** [comment]
Yes! Running 11.0.1 right now. Installed Catalina and ran the Installer with zero issues. Make sure you Time Machine first

**Comment by alfonsodev** [comment]
what Mac version do you have Config.plist ? I'm using `MacBookPro10,2` as indicated in the `README.md` but the update is not available.

**Comment by alfonsodev** [comment]
OK, I run GenSMBIOS again with `MacBookPro11,5` which is a mid 2015 that supports Big Sur, and now I can download from the App Store. 🤞

**Comment by mave-panda** [comment]
> what Mac version do you have Config.plist ? I'm using `MacBookPro10,2` as indicated in the `README.md` but the update is not available. 

I also have mine set to MacBookPro10,2 but I downloaded 11.0.1 using gib and then ran the pkg under Catalina. It built the installer app straight to Applications and I ran it from there. Everything runs just fine except for Wifi & Bluetooth (waiting on my new card)

There is one bug with the trackpad where it'll stop responding but putting the machine into Sleep and powering back on resolves it

Edit: forgot to say I'm running the stock firmware with this

**Comment by alfonsodev** [comment]
I just want to report that the installation process worked 👌
>Everything runs just fine except for Wifi & Bluetooth (waiting on my new card).  

@mave-panda I'm curious what card model will you use, I'm looking for one myself.

**Comment by mave-panda** [comment]
> I just want to report that the installation process worked 👌

CONGRATS!

> @mave-panda I'm curious what card model will you use, I'm looking for one myself.

Same card mentioned by OP; Azurewave AW-CB160H BCM94360HMB. If you'd like to see what else is out there try search "Hackintosh wifi" on eBay and Amazon

Edit: to add I also decided to spring for a 9-cell battery as well (50 USD on eBay) since the stock battery doesn't last more than 15 minutes in standby

**Comment by banhbaoxamlan** [comment]
I tried Big Sur and I not recomemed :(, performance in Big Sur was very poor.

**Comment by 13631181823** [comment]
> I tried Big Sur and I not recomemed :(, performance in Big Sur was very poor.

bad experience in bigsur public，so i quit to using win.🌚🌚

**Comment by mave-panda** [comment]
> I tried Big Sur and I not recomemed :(, performance in Big Sur was very poor.

So strange 🤔 It's been fine for me so far, with my 180GB SSD and only 8GB of memory and a slightly slower CPU than yours at 2.6GHz ![image](https://user-images.githubusercontent.com/61158973/101284958-2ab64d80-37b9-11eb-8bec-ac245c36c097.png)

**Comment by 13631181823** [comment]
i tryed 11.1 today,i think it just better than 11.0.1.🐂🍺

**Comment by mave-panda** [comment]
> @13631181823 i tryed 11.1 today,i think it just better than 11.0.1.🐂🍺

What MacBook Pro version are you using? I've been waiting on dosdude1 to release an installer patch for Big Sur to try 11.1 as Software Update doesn't see it as an available update (EOL for the 10,2)

**Comment by oranggila** [comment]
clean install big sur 11.1, everything works like catalina but... performance not acceptable for me so lag just mounting the dmg, browsing, etc just like using atom processor with windows 10, even I turn off the pm.aml

going back to catalina, stick with it until I sold this machine 😋
my x230 
x230 i5 3320m
samsung 850evo 500gb
16gb ddr3 no name brand

**Comment by Karleow** [comment]
Installed 11.2, upgraded to 11.2.3, everything works good. Can feel UI runs a little more lag than 10.15, but use less memory. No other obvious different.

X230 3320M, 12GB ram.

**Comment by Valeria** [comment]
I'm also running 11.2.3 and performance seems very similar to Catalina. Would recommend updating if you prefer Big Sur UI or want to run updated software requiring 11.x, or stay on Catalina if you like it's UI more.

Fonts do seem to be less blurry on my 1080p screen on Big Sur tho, so that might be a reason to upgrade.

![image](https://user-images.githubusercontent.com/77569323/116126188-a3fe0080-a6c6-11eb-9483-ce089746d55c.png)
